Pectra: Hive: UnsatisfiedLinkError

Many test of the KZG point evaluation precompile Hive tests are failing with an error message similar to the following.

```
2025-02-02 08:54:59.798+00:00 | main | ERROR | BlocksSubCommand | Unable to import blocks from /blocks/0001.rlp

java.util.concurrent.CompletionException: java.lang.UnsatisfiedLinkError: 'boolean ethereum.ckzg4844.CKZG4844JNI.verifyKzgProof(byte[], byte[], byte[], byte[])'

	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)

	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)

	at java.base/java.util.concurrent.CompletableFuture.biRun(CompletableFuture.java:1453)

	at java.base/java.util.concurrent.CompletableFuture$BiRun.tryFire(CompletableFuture.java:1431)

	at java.base/java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482)

	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)

	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)

	at java.base/java.lang.Thread.run(Thread.java:1583)

Caused by: java.lang.UnsatisfiedLinkError: 'boolean ethereum.ckzg4844.CKZG4844JNI.verifyKzgProof(byte[], byte[], byte[], byte[])'

	at ethereum.ckzg4844.CKZG4844JNI.verifyKzgProof(Native Method)
```

There seem to be an issue with the native KZG library that needs to be fixed.

This issue is part of #8177

--- a/besu/src/main/java/org/hyperledger/besu/cli/subcommands/blocks/BlocksSubCommand.java
+++ b/besu/src/main/java/org/hyperledger/besu/cli/subcommands/blocks/BlocksSubCommand.java
@@ -34,6 +34,7 @@ import org.hyperledger.besu.ethereum.chain.Blockchain;
 import org.hyperledger.besu.ethereum.core.ImmutableMiningConfiguration;
 import org.hyperledger.besu.ethereum.core.ImmutableMiningConfiguration.MutableInitValues;
 import org.hyperledger.besu.ethereum.core.MiningConfiguration;
+import org.hyperledger.besu.evm.precompile.KZGPointEvalPrecompiledContract;
 import org.hyperledger.besu.metrics.MetricsService;
 import org.hyperledger.besu.metrics.prometheus.MetricsConfiguration;
 
@@ -206,6 +207,7 @@ public class BlocksSubCommand implements Runnable {
       }
       LOG.info("Import {} block data from {} files", format, blockImportFiles.size());
       final Optional<MetricsService> metricsService = initMetrics(parentCommand);
+      KZGPointEvalPrecompiledContract.init();
 
       try (final BesuController controller = createController()) {
         for (final Path path : blockImportFiles) {

